charts/spire/charts/spire-server/templates/issuer.yaml

```
@@ -6,7 +6,7 @@ Configuring CA Issuer: https://cert-manager.io/docs/configuration/ca/
 apiVersion: cert-manager.io/v1
 kind: Issuer
 metadata:
-  name: {{ include "spire-server.fullname" $ }}-selfsigned
+  name: {{ default (include "spire-server.fullname" $) .issuer_name }}
```

This will break existing users of the chart that are using the config without issuer_name. Its recent enough though its likely there are very few doing that....

It will also break on this though I think which is a show stopper?:
https://github.com/spiffe/helm-charts/pull/378/files#diff-c63b6ac3d0fed523e732a68f94aa4bc6ffc1b54e2fe15f2c9aaf8c8d7ecd5eedR48

Maybe we do something like:
```suggestion
  name: {{ printf "%s-selfsigned" (default (include "spire-server.fullname" $) .issuer_name) }}
```

?

---

When you do a helm upgrade, it deletes/creates objects with name changes. So nobody will be broken even if they're on 0.10.1. I think the name is conflicting so Ill add the selfsigned suffix

---

That's the concern. Their existing working ca will get nuked? Not sure what affect that will have on their workload that already has certs but suspect it breaks them?